lib/openstudio/workflow/jobs/run_energyplus/run_energyplus.rb

```diff
@@ -117,6 +112,45 @@ def perform
 
   private
 
+  # Look for the location of EnergyPlus
+  def find_energyplus
+    if ENV['ENERGYPLUSDIR']
+      return ENV['ENERGYPLUSDIR']
+    elsif ENV['RUBYLIB'] =~ /OpenStudio/
+      path = ENV['RUBYLIB'].split(':')
+      path = File.dirname(path.find{|p| p =~ /OpenStudio/})
+      # Grab the version out of the openstudio path
+      path += '/sharedresources/EnergyPlus-8-2-0'
+      @logger.info "found EnergyPlus path of #{path}"
+      return path
+    else
+      if /cygwin|mswin|mingw|bccwin|wince|emx/ =~ RUBY_PLATFORM
+        energyplus_path = 'C:/EnergyPlus-8-2-0'
+      else
+        energyplus_path = '/usr/local/EnergyPlus-8-2-0'
+      end
+
+    end
+  end
+
+  def prepare_energyplus_dir
+    def copy_if_exists(from, to_dir)
+      @logger.info "Copying #{from} to #{to_dir}"
+      FileUtils.copy(from, "#{to_dir}/#{File.basename(from)}") if File.exists?(from)
+    end
+
+    @logger.info "Copying EnergyPlus files to run directory: #{@run_directory}"
+    copy_if_exists("#{@options[:energyplus_path]}/libbcvtb.so", @run_directory)
```

Comment:
@nllong maybe we want to look into the new command line interface options for E+ 8.3, might be able to get rid of some of this crap